models/balancer/balancer_bpt_prices.sql

```diff
@@ -0,0 +1,32 @@
+{{ config(
```

Comment:
this one is fine to leave as a view, as all the upstream spells are materialized. just keep in mind, if you end up querying this and it still feels slower than expected, you have the option to revisit and materialize incremental too 🤝